src/step/assemble_stiffness_matrix_springs.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Profiling_ScopedRegion.hpp>
+
+#include "src/elements/springs/springs.hpp"
+
+namespace openturbine {
+
+inline void AssembleStiffnessMatrixSprings(const Springs& springs) {
+    auto region = Kokkos::Profiling::ScopedRegion("Assemble Springs Stiffness");
+    Kokkos::parallel_for(
+        "AssembleSpringsStiffness", springs.num_elems,
+        KOKKOS_LAMBDA(const int i_elem) {
+            // Apply stiffness terms with appropriate signs for each node pair
+            static constexpr std::array<std::array<double, 2>, 2> node_signs = {{
+                {1., -1.},  // Signs for node 1
+                {-1., 1.}   // Signs for node 2
+            }};
```

Comment:
Don't use std::array on device.  Also, don't use nested data structures in performance critical areas.